test/user-test.js

```diff
@@ -1,61 +1,4503 @@
 import { expect } from 'chai';
 
 import User from '../src/user';
-import data from '../src/data/users-data';
 
 describe('User', function() {
   let user;
-  let userInfo;
-  let recipe;
+  let recipes;
 
   beforeEach(function() {
-    userInfo = data[0];
-    user = new User(userInfo)
+    let sampleUserData = [
```

Comment:
This is a TON of sample data. I think we can shorten this considerably: 
- I'm sure user 2 and 3 is lovely, but honestly user 1 is all we need. 
- Also, can we just be a broke college student? Why do they have 3000 foods. Have them live off of only pancakes, and their pantry reflect that.

Reply:
Love it. I'll shorten the sample data down to one user.

Reply:
These changes were made and the tests still run.